aten/src/ATen/native/cuda/AveragePool3d.cu

```diff
@@ -29,7 +29,7 @@ __global__ void avg_pool3d_cuda_update_output(
   int dT, int dH, int dW,
   int padT, int padH, int padW,
   bool count_include_pad,
-  int offsetZ)
+  int offsetZ, int divisor_override)
```

Comment:
I think I asked this before, but is there a reason that optional doesn't work herE?

Reply:
it's not allowed to call a host function c10::optional::value from a global function

Reply:
ah, you might be able to fix that by making value a C10_HOST_DEVICE function, but this seems fine.